src/keys/get-management-token.ts

```diff
@@ -142,8 +139,12 @@ export const createGetManagementToken = (
  * @category Keys
  */
 export const getManagementToken = (privateKey: string, opts: GetManagementTokenOptions) => {
-  return createGetManagementToken(createLogger({ filename: __filename }), createHttpClient())(
-    privateKey,
-    opts
-  )
+  if (opts.reuseToken || opts.reuseToken === undefined) {
+    defaultCache = new NodeCache()
```

Comment:
I am a bit confused by this. Doesn't it mean that we are initialising the cache each time you a user tries gets a token?

Reply:
Yes you are right, I added a check that this only happens once